### PR TITLE
prune stream registry functions

### PR DESCRIPTION
**Pull request by giuseppecrj**

### CODE REVIEW

File: contracts/src/river/registry/facets/stream/StreamRegistry.sol [hunk 1]
```diff
@@ -285,70 +281,4 @@ contract StreamRegistry is IStreamRegistry, RegistryModifiers {
...
-    for (uint256 i = 0; i < streamCount; ++i) {
-      bytes32 streamId = streamIds[i];
-      Stream storage stream = ds.streamById[streamId];
-      if (stream.nodes.length == 0) continue;
-      streams[foundCount++] = StreamWithId({id: streamId, stream: stream});
-    }
-    return (foundCount, streams);
-  }
-
-  function getStreamsOnNode(
```
Comment (shuhuiluo) [inline contracts/src/river/registry/facets/stream/StreamRegistry.sol:303]: I thought `getStreamsOnNode` will be used.

Comment (giuseppecrj) [reply contracts/src/river/registry/facets/stream/StreamRegistry.sol:303]: nope

Comment (giuseppecrj) [reply contracts/src/river/registry/facets/stream/StreamRegistry.sol:303]: getStreamCountOnNode is

Comment (shuhuiluo) [reply contracts/src/river/registry/facets/stream/StreamRegistry.sol:303]: Isn't it supposed to make node sync faster?

Comment (giuseppecrj) [reply contracts/src/river/registry/facets/stream/StreamRegistry.sol:303]: I'm removing all functions we agreed on removing, if we need add new ones that are optimized we will add them back

**Review by shuhuiluo — APPROVED** [review]

**Pull request merged**